src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -68,7 +68,7 @@ You can include your DSN directly in these two files, or provide it in either of
 
 </Alert>
 
-## Extend Default Webpack Usage
+## Use `withSentryConfig`
```

Comment:
I rather titles focus on the goal over what you need to do. Before the change, the title was telling you what the section is about; after the change, you have to guess it. Wdyt?

Reply:
Fair point. I changed it to `Extend Next.js Configuration`, since that makes it immediately clear where they're supposed to make the changes.